redirections.yml

```diff
@@ -66,6 +66,9 @@ redirections:
   - from: koncorde
     to: core/1/guides/cookboks/realtime-api
 
+  - from: koncorde
```

Comment:
How does that work: won't this overwrite the [previous rule](https://github.com/kuzzleio/documentation/blob/bae2dcdad018ba786584cfb8254bae0c642bca5c/redirections.yml#L66)?